modular_chomp/code/modules/mob/living/silicon/robot/sprites/syndicate.dm

```diff
@@ -0,0 +1,72 @@
+/datum/robot_sprite/dogborg/syndie/v2dogborgsyndie
```

Comment:
This V2 sprite was exclusively meant and is in use for the combat medic.

Reply:
deleting